src/rust/ide/src/model/project/synchronized.rs

```diff
@@ -98,40 +99,31 @@ impl ExecutionContextsRegistry {
 pub struct ProjectManagerUnavailable;
 
 
-
 // === Data ===
 
-/// A structure containing the project's unique ID and name.
+/// A structure containing the project's properties.
+#[allow(missing_docs)]
 #[derive(Debug,Clone)]
-pub struct Data {
+pub struct Properties {
```

Comment:
Perhaps its Flowbox legacy, but seeing `properties` used caused me think that this is something totally different.